oracle-linux-image-tools/cloud/vagrant-virtualbox/image-scripts.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+#
+# Cleanup and package image for the "vagrant-virtualbox" image
+#
+# Copyright (c) 1982-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This copyright line needs to be updated.

Reply:
Updated -- bf5ca17